pkgs/top-level/all-packages.nix

```diff
@@ -23582,17 +23582,17 @@ with pkgs;
 
   wcslib = callPackage ../development/libraries/science/astronomy/wcslib { };
 
-  webkitgtk = callPackage ../development/libraries/webkitgtk {
```

Comment:
Now the first commit does not evaluate. It references `webkitgtk_4_0` but it is only introduced in the second one.

Reply:
First commit is now introducing both webkitgtk_4_0 and aliasing on all-packages.nix webkitgtk to webkitgtk_4_0.

Then treewide changes

Finally moving webkitgtk alias into aliases with a warning.